deployment/api_transparency_dev/main.tf

```diff
@@ -204,7 +204,53 @@ resource "google_compute_url_map" "default" {
       service = google_compute_backend_bucket.firmware_artefacts_ci_1.id
     }
 
-    # TODO(prod logs & artefacts)
+    # Prod log rev 0 (wave 0 devices)
+    path_rule {
+      paths = [
```

Comment:
Hmm, maybe for a follow-up, but we probably should be able to use `for_each` or `count` here (and with the serving buckets, etc.) to make it really easy to spin up and map subsequent logs.

Reply:
Good future project, but not worth doing now.